apps/frontend/src/libs/components/slider/slider.tsx

```diff
@@ -0,0 +1,139 @@
+import ValueBubble from "~/assets/img/value-bubble.svg?react";
+import { getValidClassNames } from "~/libs/helpers/helpers.js";
+import {
+	useCallback,
+	useEffect,
+	useRef,
+	useState,
+} from "~/libs/hooks/hooks.js";
+
+import {
+	FULL_PROGRESS,
+	MAX_VALUE,
+	MIN_SCORE_VALUE,
+	MIN_VALUE,
+	RESIZE_EVENT,
+	SLIDER_BACKGROUND_COLOR,
+} from "./libs/constants/constants.js";
+import { formatToKebabCase } from "./libs/helpers/helpers.js";
+import styles from "./styles.module.css";
+
+type Properties = {
+	id: number;
+	label: string;
+	onValueChange: (categoryId: number, value: number) => void;
+	value: number;
+};
+
+const Slider: React.FC<Properties> = ({
+	id,
+	label,
+	onValueChange,
+	value,
+}: Properties) => {
+	const [sliderValue, setSliderValue] = useState<number>(value);
+	const [step, setStep] = useState<number>(value);
+	const rangeReference = useRef<HTMLInputElement>(null);
+	const bubbleReference = useRef<HTMLDivElement>(null);
```

Comment:
```suggestion
	const rangeReference = useRef<HTMLInputElement | null>(null);
	const bubbleReference = useRef<HTMLDivElement | null>(null);
```
Let's be more specific